tcipostman/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask_mail import Mail, Message
+from tcidatabase import models
+from tcidatabase import db
+
+
+def init_app(app):
+    Postman.app = app
+    Postman.mail = Mail(app)
+    db.signals.post_save.connect(Postman.send_client_token,
+                                 sender=models.SingleToken)
+
+
+class Postman:
+    app = None
+    mail = None
```

Comment:
Is [the setup in readme](https://github.com/anthropedia/tci-postman/pull/1/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R18) replying to your point?